### PR TITLE
Rooster 6.3.4

### PR DESCRIPTION
**Pull request by JiuqingSong**
1. Fix clear format behavior for IE and Edge
2. Fix Indent command to remove top and bottom margin.

### CODE REVIEW

File: packages/roosterjs-editor-api/lib/format/clearFormat.ts [hunk 1]
```diff
@@ -1,11 +1,25 @@
 import execFormatWithUndo from './execFormatWithUndo';
-import { Editor } from 'roosterjs-editor-core';
+import { Editor, browserData } from 'roosterjs-editor-core';
+import setFontName from './setFontName';
+import setFontSize from './setFontSize';
+import setTextColor from './setTextColor';
+import setBackgroundColor from './setBackgroundColor';
```
Comment (MSJiaGuo) [inline packages/roosterjs-editor-api/lib/format/clearFormat.ts]: sort imports?

File: packages/roosterjs-editor-api/lib/format/clearFormat.ts [hunk 2]
```diff
@@ -1,11 +1,25 @@
...
 export default function clearFormat(editor: Editor): void {
     editor.focus();
     // We have no way if this clear format will really result in any DOM change
     // Let's just do it with undo
     execFormatWithUndo(editor, () => {
-        editor.getDocument().execCommand('removeFormat', false, null);
+        editor.runWithoutAddingUndoSnapshot(() => {
+            editor.getDocument().execCommand('removeFormat', false, null);
+
+            if (browserData.isEdge || browserData.isIE) {
```
Comment (MSJiaGuo) [inline packages/roosterjs-editor-api/lib/format/clearFormat.ts]: Only for IE and Edge? How about other browsers?

File: packages/roosterjs-editor-api/lib/format/setIndentation.ts [hunk 3]
```diff
@@ -1,11 +1,21 @@
 import execFormatWithUndo from './execFormatWithUndo';
 import { Editor } from 'roosterjs-editor-core';
 import { Indentation } from 'roosterjs-editor-types';
+import getFormatState from '../format/getFormatState';
+import queryNodesWithSelection from '../cursor/queryNodesWithSelection';
```
Comment (MSJiaGuo) [inline packages/roosterjs-editor-api/lib/format/setIndentation.ts]: sort imports?

File: packages/roosterjs-editor-api/lib/format/clearFormat.ts (hunk 2 above)
Comment (JiuqingSong) [reply packages/roosterjs-editor-api/lib/format/clearFormat.ts]: Other browsers can already handle this well.

File: packages/roosterjs-editor-api/lib/format/clearFormat.ts (hunk 1 above)
Comment (JiuqingSong) [reply packages/roosterjs-editor-api/lib/format/clearFormat.ts]: fixed

File: packages/roosterjs-editor-api/lib/format/setIndentation.ts (hunk 3 above)
Comment (JiuqingSong) [reply packages/roosterjs-editor-api/lib/format/setIndentation.ts]: fixed

**Pull request merged**